src/pages/resultsView/enrichments/AlterationEnrichmentsTable.tsx

```diff
@@ -120,17 +121,25 @@ export default class AlterationEnrichmentTable extends React.Component<IAlterati
 
         columns[AlterationEnrichmentTableColumnType.P_VALUE] = {
             name: "p-Value",
-            render: (d: AlterationEnrichmentRow) => <span style={{whiteSpace: 'nowrap'}}>{toConditionalPrecision(d.pValue, 3, 0.01)}</span>,
-            tooltip: <span>Derived from Fisher's exact test</span>,
-            sortBy: (d: AlterationEnrichmentRow) => d.pValue,
+            render: (d: AlterationEnrichmentRow) => {
+                const pValue = d.pValue == undefined ? 'NA' : toConditionalPrecision(d.pValue, 3, 0.01);
+                return <span style={{ whiteSpace: 'nowrap' }}>{pValue}</span>;
+            },
+            tooltip: <span>Derived from {this.props.groupsCount > 2 ? "Chi-Square" : "Fisher's exact"} test</span>,
+            // set to 0 when p-value is undefined. this happens when all the cases in all groups are altered
+            sortBy: (d: AlterationEnrichmentRow) => d.pValue || 0,
```

Comment:
just confirming it's intended that now these rows will appear at the top when sorting in ascending order of p-value?

Reply:
yes, this is to indicate all the cases are altered in all groups